build/dependencies.props

```diff
@@ -178,7 +178,7 @@
     <NETStandardLibrary20PackageVersion>2.0.1</NETStandardLibrary20PackageVersion>
     <NewtonsoftJsonBsonPackageVersion>1.0.1</NewtonsoftJsonBsonPackageVersion>
     <NewtonsoftJsonPackageVersion>10.0.1</NewtonsoftJsonPackageVersion>
-    <OracleManagedDataAccessPackageVersion>12.2.1100</OracleManagedDataAccessPackageVersion>
+    <OracleManagedDataAccessCorePackageVersion>2.12.0-beta2</OracleManagedDataAccessCorePackageVersion>
```

Comment:
@Eilon - 3rd party dependency update. Used in sample provider only. Non-shipping package.

Reply:
Working on it

Reply:
Chatted with @smitpatel and @pakrym . This is a proprietary license so I'll have to review more closely when I'm back at the office on Monday.